GSoC-2025/Prisha_Sharma.md

```diff
@@ -77,74 +75,99 @@ This project aims to enhance CellFinder by adding multi-channel support for brai
       <td>15</td>
     </tr>
     <tr>
-      <td><b>Week 1-2</b><br>Focus: Integrating attention into blob detection</td>
+      <td><b>Week 1</b><br>Modify cell candidate detection for Multichannel detection</td>
       <td>
         <ol type="1">
-          <li>Modify the blob detection pipeline to process arbitrary channels through attention.</li>
-          <li>Conduct multi-channel blob detection tests</li>
+          <li>Modify the cell candidate detection pipeline to handle N channels.</li>
+          <li>Feature Extraction for Each Channel</li>
+          <li>Normalize feature values across different channels </li>
+          <li>Combine extracted features from all channels to create a comprehensive feature representation for each cell candidate</li>
           <li>Refine based on mentor's feedback</li>
         </ol>
       </td>
       <td>35</td>
     </tr>
     <tr>
-      <td><b>Week 3-4</b><br>Focus: Neural network integration</td>
+      <td><b>Week 2</b><br>Refine Multichannel Cell Candidate Detection</td>
       <td>
         <ol type="1">
-          <li>Integrate attention modules into the neural network classifier to classify cell candidates from brain images with an arbitrary number of channels  .</li>
-          <li>Start small-scale training experiments on multi-channel brain datasets.</li>
-          <li>Refine based on mentor's feedback</li>
+          <li>Optimize feature selection across channels for better accuracy</li>
+          <li>Improve the detection algorithm based on cell shape, intensity, and texture to reduce false positives.</li>
```

Comment:
I'm not sure what this means? Could you add more detail about your proposed implementation?

We also don't really care too much about false positives, as that's what the classification step is for.